### PR TITLE
IRAP-544 Migrate from ChoiceArrayField to ManyToManyField

### PR DESCRIPTION
**Pull request by SamDudley**
- wip
- wip

### CODE REVIEW

File: core/views/task_list.py [hunk 1]
```diff
@@ -160,9 +160,9 @@ def get(self, request, *args, **kwargs):
         return response
 
     def get_context_data(self, **kwargs):
```
Comment (marcelkornblum) [inline core/views/task_list.py:162]: do we need to keep this?

File: tests/core/test_irap_request.py [hunk 2]
```diff
@@ -345,23 +345,22 @@ def _setup(self, db, client, user, directorate):
...
+        ]
+
         r = self.client.post(
             reverse("core:request:type-of-data", args=[self.ir.pk]),
             {
-                "types_of_data": [
-                    IrapRequest.TypesOfData.PERSONAL_DATA,
-                    IrapRequest.TypesOfData.OTHER_DATA,
-                ],
+                "types_of_data": types_of_data,
```
Comment (marcelkornblum) [inline tests/core/test_irap_request.py:356]: <3

**Review by marcelkornblum — APPROVED** [review]
love it - just one Q

File: core/views/task_list.py (hunk 1 above)
Comment (SamDudley) [reply core/views/task_list.py:162]: No we don't, but as I'm about to work closely in that area I felt it OK to leave it there.

**Pull request merged**